script/get_catalog.sh

```diff
@@ -34,7 +34,7 @@ if [ ! -z $2 ]; then
 fi
 
 # Refresh catalog sets. We can clean any leftover as well.
-rm -f ${rootdir}/resources/camel-catalog-*
+rm -f ${rootdir}/pkg/resources/resources/camel-catalog-*
```

Comment:
is this `resources/resources` correct ?

Reply:
Yes. In a previous work we have moved to use `go:embed` and we kept using the same structure (config dir and resources dir) under the same directory to avoid making any more disruptive change. Ideally we'd need to normalize that to use a flat directory. But that would be another chunk of work.